govc/library/session/ls.go

```diff
@@ -102,26 +107,45 @@ func (cmd *ls) Run(ctx context.Context, f *flag.FlagSet) error {
 		if len(ids) == 0 {
 			continue
 		}
-		var sessions []*library.Session
+		i := &info{
+			Files: make(map[string]any),
+			kind:  k.kind,
+		}
 
 		for _, id := range ids {
 			session, err := k.get(ctx, id)
 			if err != nil {
 				return err
 			}
 			item, err := m.GetLibraryItem(ctx, session.LibraryItemID)
-			if err != nil {
-				return err
+			if err == nil {
+				// can only show library path if item exists
+				lib, err := m.GetLibraryByID(ctx, item.LibraryID)
+				if err != nil {
+					return err
+				}
+				session.LibraryItemID = fmt.Sprintf("/%s/%s", lib.Name, item.Name)
```

Comment:
Curious:
why use lib name and item name as the libraryItemID instead of the real UUID of the item? I'm not familiar with the API code here, I guess this is following some existing pattern in govmomi?

Reply:
In general govc uses human-readable names rather than IDs where possible. Such as using name or inventory path rather than MOID. And in the case of content library, we also support a [named path](https://github.com/vmware/govmomi/blob/main/vapi/library/finder/README.md)
But it is also possible to use the ID instead if needed.

Reply:
I see. It's just a little confusing when I see the field name indicate an ID, I guess it's fine as long as users can navigate objects following this pattern, i.e. able to find the library item given an update session ID.

Reply:
That was just re-using an existing type + field. Updated to use a helper type, so `LibraryItemID` is left as-is as you can see with the `-json` output.